qadence2_platforms/backends/_base_analog/device_settings.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from abc import ABC
+
+from pulser.devices import Device
+
+
+class DeviceSettings(ABC):
+    """
+    Device settings for easy of building sequence, register and interface logic.
+
+    name (str): the name of the device, ex: `Analog Device`
+    name_short (str): a shorter name version, ex: `analog`
+    device (pulser.devices.Device): the pulser device
+    grid_scale_range (tuple[float, float]): a tuple of min and max values for the
+        grid scale, ex: `(1.0, 1.0)` (should not scale), `(1.0, 10.0)` (up to
+        10.0, included)
+    available_grid_types (tuple[str]): a tuple of all the possible grid types
+        for the device, ex: `("triangular",)`, `("linear", "square")`
+    available_directives (tuple[str], optional): a tuple of available directives,
+        ex: `("enable_digital_analog")`
+    """
+
+    _name: str
+    _name_short: str
+    _device: Device
+    _grid_scale_range: tuple[float, float]
+    _available_grid_types: tuple[str, ...]
+    _available_directives: tuple[str, ...] | tuple | None
+
+    @property
+    def name(self) -> str:
+        return self._name
+
+    @property
+    def name_short(self) -> str:
```

Comment:
Is there a need for a short name ?

Reply:
may be useful to match the short name somewhere with the backend name or to comparison